### PR TITLE
Profile picture uploading

### PR DESCRIPTION
**Pull request by lullofthesea**

### CODE REVIEW

File: models/user.model.js [hunk 1]
```diff
@@ -1,9 +1,41 @@
-const mongoose = require('mongoose');
+const mongoose = require("mongoose");
 const Schema = mongoose.Schema;
 
-const { Array, Boolean, String } = Schema.Types;
+const { Boolean, String } = Schema.Types;
 
 const UserSchema = new Schema({
+<<<<<<< HEAD
```
Comment (aidenszeto) [inline models/user.model.js:7]: Could we get rid of the merge markers in this file? We may need to rebase since master includes `friends` as well.

Comment (aidenszeto) [reply models/user.model.js:7]: I think the user model should just be what's currently on master + the `profileImg` field you added here

File: routes/user.js [hunk 2]
```diff
@@ -89,63 +90,91 @@ router.post("/setUserInfo/:userId", async (req, res) => {
...
+    try {
+      const imageUploadRes = await uploadImagesToAWS(
+        req.files["image"],
+        "profileImg"
+      );
+      const imgUrl = imageUploadRes[0];
+
+      const userToUpdate = await User.findOneAndUpdate(
+        { _id: req.params.userId },
+        { $set: { profileImg: imgUrl } },
```
Comment (aidenszeto) [inline routes/user.js:107]: Haven't been able to get this to set on my end. The route is being called but the database document remains unchanged when I try to upload a pfp

Comment (lullofthesea) [reply routes/user.js:107]: Seems like the user model didn't get pushed.  Merged + fixed and pushed.

Comment (aidenszeto) [reply routes/user.js:107]: Awesome will test and get back tonight 👍

**Review by aidenszeto — APPROVED** [review]

**Pull request merged**